src/docs/product/performance/transaction-summary.mdx

```diff
@@ -103,7 +103,13 @@ The "Spans" tab displays a list of suspect spans that correspond to where most o
 
 Choose from one of several [metrics](#metrics) to sort spans and identify different types of problems. Filter spans to only see the span operation type you're interested in.
 
-Clicking on a span will give you more details about the specific span [group](#grouping). You can see the performance of the span over time and see a list of transaction events that contain the span.
+Clicking on a span will take you to its summary page that will give you more details about this specific span [group](#grouping). On the page you can see the performance of the span over time and see a list of transaction events that contain the span.
+
+<Note>
+
+If you're in the Early Adopter program, you can also see the distribution of the span self time. This histogram can help you identify systemic performance issues across the entire dataset by showing how self time for every span instance is distributed. You can also select a subset of the dataset by zooming into the distribution histogram which will effectively set min and max bounds. The stats and transaction events table will reflect that change and will only include spans with self time that are within the set bounds. Adding filters allows you to focus on the best or worse performing spans so you can troubleshoot performance issues more easily.
+
```

Comment:
```suggestion

### Span Summary

Clicking on a span will take you to its summary page which will give you more details about this specific span [group](#grouping). On the page, you can see the performance of the span over time and see a list of transaction events that contain the span.

<Note>

This span distribution feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

If you're in the Early Adopter program, you can also see the distribution of the span self time using the (added) "Display" dropdown. This histogram can help you identify systemic performance issues across the entire dataset by showing how self time for every span instance is distributed. 

You can also select a subset of the dataset by zooming into the distribution histogram. The stats and transaction events table will reflect that change and will only include spans with self time that are within the bounds of your subset. This allows you to focus on the best or worse performing spans so you can troubleshoot performance issues more easily.


```